bank-sdk/sdk/src/main/java/net/gini/android/bank/sdk/GiniBank.kt

```diff
@@ -54,7 +56,7 @@ object GiniBank {
     private var giniApi: GiniBankAPI? = null
 
     var digitalInvoiceOnboardingNavigationBarBottomAdapter: DigitalInvoiceOnboardingNavigationBarBottomAdapter = DefaultDigitalInvoiceOnboardingNavigationBarBottomAdapter()
-
+    var digitalInvoiceOnboardingIllustrationAdapter: DigitalInvoiceOnboardingIllustrationAdapter = ImageDigitalInvoiceOnboardingIllustrationAdapter()
```

Comment:
[Suggestion] If we make `OnboardingIllustrationAdapter` and `ImageOnboardingIllustrationAdapter` public, then we could reuse them in Bank SDK and can avoid some duplication. What do you think?
```suggestion
    var digitalInvoiceOnboardingIllustrationAdapter: OnboardingIllustrationAdapter = ImageOnboardingIllustrationAdapter(R.drawable.gbs_digital_invoice_list_image, R.string.gbs_digital_invoice_list_image_content_description)
```

Oh and we also found the content description for the image! 🙈 Please create the string resources for it and use `Digital invoice illustration` as the value for both DE and EN for now. Thank you!